### DISCUSSION TITLE
THANK YOU FOR EZDXF

### DISCUSSION THREAD

**Discussion by dancergraham** [mozman/ezdxf#730] [General]
It's great

**Comment by fastbrowncar** [comment]
I'm so grateful. I learned how to move text from text file into a dxf, at a specified location. Then I learned to read the dxf for text, print out the text, learn it's location and size. 

This is the tip of the iceberg. This tool seems so well made and actively maintained. I'm fully impressed and grateful.

**Comment by mozman** [comment]
Thank you for the nice feedback!
